hera_cal/datacontainer.py

```diff
@@ -243,22 +243,21 @@ def __sub__(self, D):
         otherwise subtract D from each ndarray in self.
```

Comment:
I'd edit the docstring to clarify that the original object is always duplicated and that the metadata of the original object is preserved (but not the new object).